pubcontrol.go

```diff
@@ -71,6 +71,13 @@ func (pc *PubControl) ApplyConfig(config []map[string]interface{}) {
 			case []byte:
 				pcc.SetAuthJwt(claim, entry["key"].([]byte))
 			}
+		} else if _, ok := entry["key"]; ok {
+			switch entry["key"].(type) {
```

Comment:
Always treating the value as a string here is probably fine per the below suggestion.